### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
Grid FTW!

## Screenshot
grid

## Trello Card
https://trello.com/c/3CrEq5IY/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,29 +70,31 @@ <h2 class="page-title">Back-End Tools</h2>
...
-        </p>
-      </div>
-      <div class="tech-container">
-        <img class="tech-logo" src="assets/django_icon.svg" alt="DJango logo">
-        <p class="tech-title">Django</p>
-        <p class="tech-sub">Release: 2005</p>
-        <p class="tech-desc">
-          Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.
-        </p>
+      <div class="tech-grid">
```
Comment (mmamaniwolox) [inline index.html]: You can remove this div and use a big grid in tech section

File: index.html [hunk 2]
```diff
@@ -25,37 +25,39 @@ <h2 class="page-title">Front-End Tools</h2>
...
-        </p>
-      </div>
-      <div class="tech-container">
-        <img class="tech-logo" src="assets/polymer_icon.svg" alt="Polymer logo">
-        <p class="tech-title">Polymer</p>
-        <p class="tech-sub">Release: 2015</p>
-        <p class="tech-desc">
-          Polymer is an open-source JavaScript library for building web applications using Web Components.
-        </p>
+      <div class="tech-grid">
```
Comment (mmamaniwolox) [inline index.html]: You can remove this div and use a big grid in tech section

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -9,82 +9,82 @@
...
+        <div class="tech-container head-tech">
+          <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS logo">
+          <p class="tech-title">ReactJS</p>
+          <p class="tech-sub">Release: 2013</p>
+          <p class="tech-desc">
+            React is declarative, component-based Javascript library for building user interfaces.
+          </p>
+        </div>
+        <div class="tech-container nth-tech">
```
Comment (mmamaniwolox) [inline index.html]: You can remove `nth-tech` from all the small techs. You can write the style in `tech-container` and you will be overriding that style in `head-tech` which is the one that needs to be different

File: scss/tech.scss [hunk 4]
```diff
@@ -2,45 +2,43 @@
...
+  justify-content: center;
+  justify-items: center;
+  margin: 50px 0px 150px;
+}
+
+.head-tech {
+  align-content: center;
+  justify-items: center; 
+  border-right: 2px solid $earls-green; 
+  display: grid;
```
Comment (mmamaniwolox) [inline scss/tech.scss:27]: I think you dont need this line, unless you are going to use grid in head-tech's childs

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/tech.scss (hunk 4 above)
Comment (rodrigoWolox) [reply scss/tech.scss:27]: Yup, i'm using this "align-content: center;  justify-items: center;" to vertical-align ".tech-title, .tech-sub, .tech-desc"

Comment (mmamaniwolox) [reply scss/tech.scss:27]: Sweet

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**